flanker/addresslib/address.py

```diff
@@ -558,7 +558,7 @@ def to_unicode(self):
         Converts to unicode.
         """
         if self.display_name:
-            return u'{} <{}@{}>'.format(self.display_name, self.local_part, self.domain)
+            return u'{} <{}@{}>'.format(smart_quote(self.display_name), self.local_part, self.domain)
```

Comment:
Sure, good idea. It touches on a few tests but none of them are obvious. I'll add a couple that are more narrowly scoped.